### ISSUE TITLE
question

### ISSUE THREAD

**Issue by playdasegunda** [jeonchangbin49/LimitAug#1]
I still don't understand how LimitAug works, is it useful for normalizing audio files in the training step of a source separation model, or should it be used as a post inference complement of an instrumental file?

**Comment by jeonchangbin49** [comment]
Hi, Lucas! Thanks for your interest on my work. Basically, the main concept of my work is reducing the domain shift of source separation models. In real world, commercial music we hear everyday is loud, but training examples of source separation  models are not (See Figure 2.) 

<img width="372" alt="image" src="https://user-images.githubusercontent.com/60494498/211122737-cd74598e-d876-4b63-a2ab-e4f20a12d8fb.png">

Also, every commercial music is a mastering-finished, limiter-applied music. Since there are some special characteristics that the usage of a limiter induces, I believed that using a limiter during the online training data construction process would be helpful and it really did.

So, LimitAug is a concept of using a limiter as data augmentation (in training), but I found that additional loudness normalization after applying the limiter was helpful for the performance. In this case, loud-norm should be also applied in an inference stage.

In training,
Input (Augmentation using limiter + loud-norm) => network => loud-normalized output

In inference,
Input (loud-norm) => network => loud-normalized output => output (increase gain by the amount of reduced in input loud-norm)

**Comment by playdasegunda** [comment]
thanks a lot for the explanation!
